app/models/concerns/normalize_city_name.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+module NormalizeCityName
+  extend ActiveSupport::Concern
+
+  included do
+    before_validation :normalize_city_name
+
+    def normalize_city_name
+      return if city.blank?
+
+      self.city = city.squish
+      # if it's only one word, make sure it's not ALLCAPS
+      self.city = if city.split(/[^[[:word:]]]+/).size == 1
+                    city.titleize
+                  else
+                    city.upcase_first
+                  end
+
+      normalized_from_yml = normalize_from_yml
+      return if normalized_from_yml
```

Comment:
This looks a bit ugly. Any suggestions how to write this in a better way?

Reply:
`normalize_from_yml or normalize_from_other_db_records`